src/libs/sfdk/utils.cpp

```diff
@@ -88,4 +88,18 @@ Utils::PortList toPortList(const QList<quint16> &portList)
     return toPortList(Utils::transform(portList, [](quint16 port) { return Port(port); }));
 }
 
+QString separator(TextStyle textStyle)
+{
+    switch (textStyle) {
+        case TextStyle::CamelCase:
+            return QStringLiteral("");
+        case TextStyle::SnakeCase:
+            return QStringLiteral("_");
+        case TextStyle::Pretty:
+            return QString("\xC2\xA0"); //NBSP
```

Comment:
- `QStringLiteral` should be used here too.
- I would use just `return {}` for the `CamelCase` case and for the unreachable return at the end
- `Pretty` should come first to match the order in which it it declared

Reply:
`QStringLiteral` don't work for NBSP is some places

Reply:
Try `QStringLiteral(u"\u00A0")` instead.

Reply:
Or may be better use `QString(QChar::Nbsp);` ?

Reply:
Great! :) And further shortened as `return {QChar::Nbsp};`

Reply:
Done